src/Toolchain/ToolchainProvider.ts

```diff
@@ -221,15 +228,15 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
       return;
     }
 
-    Logger.info(this.tag, `Compile toolchain: ${activeToolchain.info.name}-${activeToolchain.info.version?.str()}`);
+    Logger.info(this.tag, `Run config file with ${activeToolchain.info.name}-${activeToolchain.info.version?.str()} toolchain.`);
```

Comment:
I think 'running onecc' sounds to make sense than 'running config file', so I suggest:

```suggestion
    Logger.info(this.tag, `Run onecc with ${cfg} cfg and ${activeToolchain.info.name}-${activeToolchain.info.version?.str()} toolchain.`);
```